### ISSUE TITLE
Weather Underground no longer issues free API keys

### ISSUE THREAD

**Issue by stellarore** [G6EJD/WeatherUnderground-Weather-Display-on-an-ILI9341-TFT#2]
Discontinued May 15, 2018. Previous API keys are still working, but no word when that may change. People are getting quotes that the new smallest IBM package is $850/month.

https://www.wunderground.com/weather/api/
https://apicommunity.wunderground.com/weatherapi/topics/weather-underground-api-changes

**Comment by G6EJD** [comment]
Yes, we shall have to start the transfer to Openweathermap which is free. I’m stopping my Weather data upload to WU if they start charging me to read my own data.
